app/assets/stylesheets/application.css.erb

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
Once you start working in a file, feel free to remove the comments Rails gives you in a file. Keeps things cleaner.